aspnetcore/grpc/test-tools.md

```diff
@@ -19,32 +19,22 @@ Tooling is available for gRPC that allows developers to test services without bu
 
 This article discusses how to:
 
-* Download and install gRPCurl and gRPCui.
 * Set up gRPC reflection with a gRPC ASP.NET Core app.
+* Download and install gRPCurl and gRPCui.
 * Discover and test gRPC services with `grpcurl`.
 * Interact with gRPC services via a browser using `grpcui`.
 
-## About gRPCurl
-
-gRPCurl is a command-line tool created by the gRPC community. Its features include:
-
-* Calling gRPC services, including streaming services.
-* Service discovery using [gRPC reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md).
-* Listing and describing gRPC services.
-* Works with secure (TLS) and insecure (plain-text) servers.
-
-For information about downloading and installing `grpcurl`, see the [gRPCurl GitHub homepage](https://github.com/fullstorydev/grpcurl#installation).
-
-![gRPCurl command line](~/grpc/test-tools/static/grpcurl.png)
+> [!NOTE]
+> To learn how to unit test gRPC services, see <xref:grpc/test-services>.
 
 ## Set up gRPC reflection
 
-`grpcurl` must know the Protobuf contract of services before it can call them. There are two ways to do this:
+Tooling must know the Protobuf contract of services before it can call them. There are two ways to do this:
 
-* Set up [gRPC reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) on the server. gRPCurl automatically discovers service contracts.
+* Set up [gRPC reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) on the server. Tooling such as gRPCurl and Postman automatically discovers service contracts.
```

Comment:
Subject-verb agreement is fine, but it might be easier on the 👂 and loc as ...

```suggestion
* Set up [gRPC reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) on the server. Tools, such as gRPCurl and Postman, automatically discover service contracts.
```